### PR TITLE
Added GELU function into Keras activations

### PR DESCRIPTION
**Pull request by joyalbin**

### CODE REVIEW

**Comment by gbaned** [comment]
@fchollet Hi, Could you PTAL and approve.

**Comment by fchollet** [comment]
Thank you for the PR.

Our guidelines for adding new symbols in the Keras API are the following:

```
- It should be broadly useful to our users, rather than a niche feature that is only relevant to 1-2 teams at Google or to a specific vertical of researchers. Niche features should be maintained independently by those who need them (e.g. by extending the API via subclassing), as third-party add-on packages.
- It should be widely recognized as a machine learning best practice. We will not add new layers/etc that were recently published to ArXiv.org, even in case of claims of increased accuracy/etc. We only add new objects that are already commonly used in the machine learning community. Presumably, a new technique that does result in meaningful gains would be broadly adopted after a few months anyway (like ResNet), and that’s when we would be adding it to the core API. SIG-addons maintains a repository of significantly more volatile and independently maintained code to which the barriers to entry are lower.
- It should have an owner committed to maintaining it in the long term. In particular, the code should be maintainable by multiple people on the team, not just by one technical guru.
```
[source](https://docs.google.com/document/d/1uzWNPseXHywqiRyjvazY6eaxe23BEG2T82JasndCJjI/)

The symbol being considered here does not represent a current deep learning best practice, and as such we will not add it. It falls under "We will not add new layers/etc that were recently published to ArXiv.org, even in case of claims of increased accuracy/etc. We only add new objects that are already commonly used in the machine learning community".

Consider adding it to TensorFlow Add-Ons instead.
